config/config.properties

```diff
@@ -147,6 +147,24 @@ pravegaservice.dataLog.impl.name=BOOKKEEPER
 # Default value: true
 # storage.appends.enable=true
 
+# Whether the lazy commit functionality is enabled.
+# Valid values: true, false
+# Default value: true
+# storage.commit.lazy.enable=true
```

Comment:
Why should the user be able to decide this policy?
If it is expected that an user can configure this behaviour then it is better to state clearly what are the pros and cons

Reply:
removed from config.